NASA ADS 2.js

```diff
@@ -0,0 +1,185 @@
+{
+	"translatorID": "7044e7f7-2c35-4b25-9755-ee87cdb7f599",
+	"label": "NASA ADS 2",
+	"creator": "Tim Hostetler",
+	"target": "^https://ui.adsabs.harvard.edu/(search|abs)/",
+	"minVersion": "3.0",
+	"maxVersion": "",
+	"priority": 100,
+	"inRepository": true,
+	"translatorType": 4,
+	"browserSupport": "gcsibv",
+	"lastUpdated": "2019-11-06 20:31:43"
+}
+
+/*
+	***** BEGIN LICENSE BLOCK *****
+
+	Copyright © 2019 Tim Hostetler
+
+	This file is part of Zotero.
+
+	Zotero is free software: you can redistribute it and/or modify
+	it under the terms of the GNU Affero General Public License as published by
+	the Free Software Foundation, either version 3 of the License, or
+	(at your option) any later version.
+
+	Zotero is distributed in the hope that it will be useful,
+	but WITHOUT ANY WARRANTY; without even the implied warranty of
+	MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+	GNU Affero General Public License for more details.
+
+	You should have received a copy of the GNU Affero General Public License
+	along with Zotero. If not, see <http://www.gnu.org/licenses/>.
+
+	***** END LICENSE BLOCK *****
+*/
+
+function getSearchResults(doc) {
+	const results = doc.querySelectorAll("a[href$=abstract]");
+	const entries = {};
+	for (let el of results) {
+		const titleEl = el.querySelector(":scope h3");
+		if (!titleEl) {
+			continue;
+		}
+		const hrefParts = el.getAttribute("href").split("/");
+		if (hrefParts.length > 2) {
+			const identifier = hrefParts[hrefParts.length - 2];
+			entries[identifier] = ZU.trimInternal(titleEl.textContent);
+		}
+	}
+	return entries;
+}
+
+function detectWeb(doc, url) {
+	if (url.includes("/search/")) {
+		return "multiple";
+	}
+	else if (url.includes("/abs/")) {
```

Comment:
How hard would it be to detect some of the other content types here? It's not super critical to detect the right type, but if e.g. matching by the id would get us mostly there, it's worth the effort (e.g. it looks like all PhD Thesis have PhDT in the ID etc.

Reply:
Yep, I can make that change.  I know for sure we can extract `PhDT` and `MsT`